### ISSUE TITLE
using cpu and gpu simultaneously

### ISSUE THREAD

**Issue by thindata** [fireice-uk/xmr-stak-cpu#471]
So on Linux I installed via git and compiled for cpu and can mine successfully. I have now added an nivida graphics card and wish to use that as well. Can it be done from just xmr-stak-cpu version (i.e can it do both cpu and gpu if so how do I compile?) or do I need to use separate nvidia one.

I have installed nivdia CUDA.

**Comment by JollyMort** [comment]
Nope, gotta use the separate miner for GPU. They can run simultaneously.

**Comment by JollyMort** [comment]
The miner is now called xmr-stak and everything is integrated, see the [readme](https://github.com/fireice-uk/xmr-stak-cpu#xmr-stak-is-now-supporting-cpu-amd-and-nvidia-gpus-in-a-unified-miner).
